### ISSUE TITLE
Dashboard Console.Writeline

### ISSUE THREAD

**Issue by dsarfati** [OrleansContrib/OrleansDashboard#33]
Is there a way to prevent the console output? We have removed all of the Orleans log messages from the console since we added a custom TraceListener to use NLog. Once those logs were removed, I noticed that we get a console output every few seconds. I assume this is caused by the dashboard?

`2017-01-25 13:52:07.3060|INFO|Demon.Mission.Host.vshost.exe|[2017-01-25 21:52:07.304 GMT    21	INFO   	100000	ManagementGrain	127.0.0.1:11111]	GetDetailedHosts onlyActive=True`

`2017-01-25 13:52:07.3320|INFO|Demon.Mission.Host.vshost.exe|[2017-01-25 21:52:07.310 GMT    22	INFO   	100000	SiloControl	127.0.0.1:11111]	GetSimpleGrainStatistics`

**Comment by richorama** [comment]
@dsarfati this is logged by the management grain in Oreans, which is called periodically by the dashboard to get some of the statistics:

https://github.com/dotnet/orleans/blob/ccc18a820b7157295e03a801606a376ba94a7a46/src/OrleansRuntime/Core/ManagementGrain.cs#L64

Are you able to raise your logging threshold to remove the 'noise' of info messages?

**Comment by dsarfati** [comment]
@richorama I'm not worried about the fact that those are in our logs, im just trying to understand why they show up in the console at all. I added an NLog trace listener and disabled all console logs yet these still show up in the console.

**Comment by enewnham** [comment]
@dsarfati I have seen this too! I have added this to my orleans silohost which has stopped the messaging. Something to do with nowin or owin self hosting? See [http://stackoverflow.com/questions/17948363/tracelistener-in-owin-self-hosting](url)

This is essentially Orleans Dashboard adding its own Debug.TraceListener, and then outputting it to the console... Here is my temporary fix

```
var started = m_SiloHost.StartOrleansSilo( );

// Orleans Dashboard adds this trace listener in... So we gotta remove it for some reason
System.Diagnostics.Trace.Listeners.Remove( "HostingTraceListener" );
```

**Comment by richorama** [comment]
Thank @enewnham - I have added your line to the dashboard startup. Released as version 1.4.4
